tests/Platforms/AbstractMySQLPlatformTestCase.php

```diff
@@ -769,10 +795,8 @@ public function testDoesNotPropagateDefaultValuesForUnsupportedColumnTypes(): vo
         $diffTable->changeColumn('def_blob', ['default' => null]);
         $diffTable->changeColumn('def_blob_null', ['default' => null]);
 
-        $diff = (new Comparator())->diffTable($table, $diffTable);
-        self::assertNotFalse($diff);
-
-        self::assertEmpty($this->platform->getAlterTableSQL($diff));
```

Comment:
These two assertions cover the bug in the current implementation rather than the expected behavior. The original test and assertions read as follows:
1. Make some schema changes that don't change the resulting SQL.
2. Assert that the diff is not empty. This is the bug being fixed.
3. Assert that even though the diff is not empty, the resulting diff SQL is. This is a workaround of the bug.